admin-ui/src/components/add-account-dialog.tsx

```
@@ -0,0 +1,554 @@
+              clearPolling()
+              clearCountdown()
+              setError(t("accountManagement.expired"))
+              setStep("error")
+              return
+            }
+            case "pending": {
+              pollingRef.current = setTimeout(() => {
+                void poll()
+              }, pollIntervalMs)
+            }
+          }
```

ESLint

[error] Expected a default case.

(default-case)